ai/STA/Tactic/tactic.py

```diff
@@ -109,12 +118,18 @@ def _find_best_next_target(self, area: Area, old_target_position, player_positio
                 return intersections[1]
 
     def debug_cmd(self):
-        #return [DebugCommandFactory().area(area) for area in self.forbidden_areas]
-        return []
+        cmds = []
+        [cmds.extend(DebugCommandFactory().area(area, color=VIOLET)) for area in self.forbidden_areas]
```

Comment:
The list should be inside the extend method. it can take an iterable to extend the list.

Reply:
Does not work

Reply:
Literable est la liste de ce qui est extend, pas la liste de liste

Reply:
`cmds.extend([DebugCommandFactory().area(area, color=VIOLET) for area in self.forbidden_areas])` ?

Reply:
Ah je comprends